packages/astro/src/cli/index.ts

```diff
@@ -113,7 +113,7 @@ async function printInfo({
 	} catch (_e) {}
 	console.log();
 	printRow('Astro version', `v${ASTRO_VERSION}`);
-	printRow('Package manager', packageManager.name);
+	printRow('Package manager', packageManager?.name);
```

Comment:
Forgot to put a note. This was needed because running `node /path/to/astro.js` directly yields no package manager, so adding the `?.` to prevent an error while running the benchmark.

Reply:
We could fix it by adding a "Could not determine." label, I'll send a PR